### ISSUE TITLE
Issue with form layout

### ISSUE THREAD

**Issue by mmartyniuk** [ant-design/ant-design#10954]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.5.3

### Environment
Windows 10, Chrome 67.0.3396.87

### Reproduction link
[https://ant.design/components/form/](https://ant.design/components/form/)

### Steps to reproduce
Go to "Registration" form, click "Register" button, enter/remove some value in "Password" field, [note labels positions below, (for example Captcha)](https://im3.ezgif.com/tmp/ezgif-3-95164aa1f3.gif).

### What is expected?
Labels should have static position

### What is actually happening?
Labels are jumping.

**Comment by mmartyniuk** [comment]
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/13848380/41602242-049f69fc-73e3-11e8-95ad-ecee6c042fbc.gif)
Attached related gif file.

**Comment by mmartyniuk** [comment]
https://drive.google.com/file/d/1gJT9N-cXyEoan_A-wAn4CrkC70eui93c/view

**Comment by spideeee** [comment]
This is not the issue with the code, AFAIK it happens only on Google Chrome, where some texts flicker by mostly a pixel up/down when other DOM elements are updated.
In my 4 years of carrier I have found many such cases (most of them were Canvas vs Div, when a canvas was updated/drawn, neighbour div element(s) tends to flicker) but only for Chrome and no possible solution. This does/may not happen in any other browser.

According to me, this is an **invalid** issue.
